src/devices/Display/README.md

```diff
@@ -36,6 +36,14 @@ display.Dots = Dot.DecimalPoint;
 display.Flush();
 ```
 
+## 14-Segment Display
+
+This [display](https://shop.pimoroni.com/products/four-letter-phat?variant=39256047178) is good for showing alpha-numeric output, and its additional segments provide a wider range of characters
+
+<img src="https://shop.pimoroni.com/cdn/shop/products/Four_letter_pHAT_5_of_5_1a005b45-151c-4938-8610-8ec758b4182d_1500x1500.JPG?v=1539263861"  width ="250px" title="Pimoroni Four-Letter Phat" />
```

Comment:
please no HTML code in Markdown files.
```suggestion
![Pimoroni Four-Letter Phat](https://shop.pimoroni.com/cdn/shop/products/Four_letter_pHAT_5_of_5_1a005b45-151c-4938-8610-8ec758b4182d_1500x1500.JPG?v=1539263861)
```

Reply:
It is not possible to resize the image using markdown as I understand it, and without resizing the image is too large. There is also already and image tag in this document (line 11) presumably for the same reason.

Any other suggestions as to how to resolve ?

Reply:
ok, then. In general, idea is to remove any html tag from the Markdown because they are processed later to generate the documentation and it creates html but, once generated, the tags have a specific css applied. Meaning, if you place some, there are high chances that the style won't apply and in the destination page it won't look good at all.

I'm ok for the exception for this one as indeed there is another one like this and we will see how it looks like once the docs will be built.